README.md

```diff
@@ -161,6 +161,11 @@ knife ec2 server create -I ami-d0f89fb9 --ssh-key your-public-key-id -f m1.mediu
 # A Windows instance via the WinRM protocol -- --ssh-key is still required due to EC2 API operations that need it to grant access to the Windows instance
 # `--spot-price` option lets you specify the spot pricing
 knife ec2 server create -I ami-173d747e -G windows -f m1.medium --user-data ~/your-user-data-file -x '.\a_local_user' -P 'yourpassword' --ssh-key your-public-key-id --spot-price price-in-USD
+
+# Pass --server-connect-attribute if the machine is in VPC mode
+# Possible values of --server-connect-attribute: private_dns_name, private_ip_address, public_dns_name, public_ip_address
+# If --server-connect-attribute is not specified, then EIP is used as External IP
+knife ec2 server create -I ami-173d747e --ssh-key your-public-key-id --ssh-user ubuntu --bootstrap-protocol ssh --subnet your-subnet --security-group-ids your-security-group-id --associate-public-ip --server-connect-attribute public_ip_address
```

Comment:
this is kind of an intimidatingly long example. I'd remove the options we don't need and shorten the ones that aren't relevant to --server-connect-attribute, e.g.

```knife ec2 server create -I ami-173d747e -x ubuntu --server-connect-attribute public_ip_address```

We can presume in this example that subnet, ssh key, and security group are relatively static for the user and set in the knife.rb.